rpc_jobs/lem_multi_node_aio.yml

```diff
@@ -0,0 +1,135 @@
+- project:
+    name: 'LEM-Multi-Node-AIO-Jobs'
+    # Note: branch is the branch for periodics to build
+    #       branches is the branch pattern to match for PR Jobs.
+    series:
+      - mitaka:
+          branch: mitaka-13.1
```

Comment:
This should be named RPC_BRANCH but I realise this is following a pattern so have created rcbops/u-suk-dev#1585 to address that.